### PR TITLE
Finished agent registered bot

### PR DESCRIPTION
**Pull request by Tdrachev**
Finished agent assigned bot

### CODE REVIEW

**Review by dikel — APPROVED** [review]
LGTM

File: forta-agents-linked/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Detects if a FORTA Agent has been assigned in a predefined threshold
+
+## Supported Chains
+
+- Polygon
+
+## Alerts
+
+- FORTA-AGENT-ASSIGNED
```
Comment (stoqnkpL) [inline forta-agents-linked/README.md:13]: We should change `agent` with `bot` everywhere `agent` is used

Comment (Tdrachev) [reply forta-agents-linked/README.md]: Done

**Review by stoqnkpL — APPROVED** [review]
LGTM

**Pull request merged**